plugins/linux-pipewire/data/locale/en-US.ini

```
@@ -1,3 +1,5 @@
+PipeWireCamera="Camera (PipeWire)"
```

I don't know if it's a good idea, but I think about removing "(PipeWire)" from this source name since no other source share the same.
The underlying technology is the future/actual API (Portals+PipeWire) to use for cameras on Linux,  I think this is unnecessary to show it.

Unlike the V4L2, actually the name is align the naming with the portal that does cameras and capture cards.

```suggestion
PipeWireCamera="Camera"
```

> Mark it as experimental, somehow

Application audio capture on Windows had "(BETA) in it's name until 30, something similar might be an idea.
```suggestion
PipeWireCamera="Camera (Experimental)"
```

---

I've renamed it to "Video Capture Device (PipeWire) (BETA)" so as to match what other sources did